packages/electron-chrome-web-store/src/browser/api.ts

```diff
@@ -298,6 +328,13 @@ export function registerWebStoreApi(webStoreState: WebStoreState) {
 
   handle('chrome.management.setEnabled', async (event, id, enabled) => {
     // TODO: Implement enabling/disabling extension
+    if (webStoreState.customSetExtensionEnabled) {
+      const details: ExtensionStatusDetails = {
+        session: webStoreState.session,
+        extensionsPath: webStoreState.extensionsPath,
+      }
+      await webStoreState.customSetExtensionEnabled(id, details, enabled)
```

Comment:
Rename to `setExtensionEnabled` and add documentation referencing [chrome.management.setEnabled](https://developer.chrome.com/docs/extensions/reference/api/management#method-setEnabled)